### PR TITLE
Mobile and desktop version capstone project - 1

### PR DESCRIPTION
**Pull request by c3c1l1a**
## Proposed changes

The aim of this pull request is to implement a mobile and desktop version of this [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)

**The following features were added**

- Personalize content  that is different from the  one in the [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
-   You should follow these [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) design guide, including:
    -   Colors.
    -   Typography: font face, size and weight.
    -   Layout: composition and space between elements.
- It should be almost identical to the original design with small adjustments like text or image.
- Two pages were built:
    -   The _home page_.
    -   The _about page_.
-   Each of these pages has versions for 2 different screen sizes:
    -   Mobile: up to 768px wide.
    -   Desktop: 768px or wider.
-   Interactions
    -   Links
        -   The _home page_ should have a link in the menu to the _about page_.
        -   The logo in the header links to the _home page_.
    -   Mobile menu
        -   When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
        -   There are no guidelines for the mobile menu to be consistent with the design (colours, typography, spacing, etc.).
        -   The mobile menu should have a close (X) button that closes the menu.
-   Dynamic page
    -   The section "Featured speakers" is created dynamically in JavaScript.
    -   A JavaScript variable with the data about the speakers is used when the page loads to create the HTML for this section dynamically.

## Type of change

- [ ] New feature 

## Checklist
- [ ] [Linting tests](https://github.com/microverseinc/linters-config) pass locally with my changes
- [ ] Used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)
- [ ] Documentation is done in a [professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  <header>
+    <nav>
+      <img class="logo" src="assets/icons/logo.svg" alt="Logo"> <!-- Placeholder logo from https://logoipsum.com-->
+      <i class="fa fa-bars fa-lg js-humburger-menu-icon"></i>
+      <ul class="menu">
+        <li><a class="button--secondary js-home-page-link" href="#">Home</a></li>
+        <li><a class="button--secondary js-about-page-link" href="#">About</a></li>
+      </ul>
+      <a class="button--primary" href="#">Book a date</a>
+    </nav>
```
Comment (phelian23) [inline index.html]: - [x] Kindly add the top short dark bar section that contains the social media links, logout, and language links.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+          <img class="js-speaker-pic" src="" alt="Image of speaker"> <!-- Placehoder images from https://i.pravatar.cc-->
+        </div>
+        <div class="featured-speakers__card__details">
+          <h3 class="heading--secondary js-speaker-name"></h3>
+          <h5 class="heading--secondary underline js-speaker-specialty"></h5>
+          <p class="paragraph-normal js-speaker-desription">With decade of experience in yoga, Dewi will guide you through this journay</p>
+        </div>
+      </div>
+  </template>
+
```
Comment (phelian23) [inline index.html:198]: - [x] Kindly add the footer of the homepage and the about page to the respective page

File: README.md [hunk 3]
```diff
@@ -1 +1,62 @@
...
+
+To get a local copy up and running follow these simple steps.
+
+### [](https://github.com/c3c1l1a/c3c1l1a.github.io#prerequisites)Prerequisites
+
+-   Any modern browser (e.g Google Chrome)
+-   Text editor (e.g [Visual Studio Code](https://code.visualstudio.com/))
+
+### [](https://github.com/c3c1l1a/c3c1l1a.github.io#setup)Setup
+
```
Comment (phelian23) [inline README.md:26]: - [x] Kindly record a video presenting the features of the project.
- [x] Also deploy the page with gh pages, and add the link to your README.md

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @c3c1l1a, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights
✅ Linters are passing 👍
✅ Good UI design 👍
✅ Good commit messages 👏
✅ Good PR description 💯

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 3 above)
Comment (mateo951) [reply README.md:26]: Addressed 🔥

File: index.html (hunk 2 above)
Comment (mateo951) [reply index.html:198]: Addressed ✔

File: index.html [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+<body>
+  <header>
+    <nav>
+      <div class="social-links">
+        <a href="https://www.facebook.com/BaliSpiritFestival/"><i class="fa fa-facebook" aria-hidden="true"></i></a>
+        <a href="https://twitter.com/balispiritfest"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+        <a href="#">Logout</a>
+        <a href="#">My page</a>
+        <a href="#">Indonesian</a>
+      </div>
```
Comment (mateo951) [inline index.html:19]: Addressed 🔥

**Review by mateo951 — APPROVED** [review]
### Hi @c3c1l1a , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 📝 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**